tests/unit/test_e2e.py

```diff
@@ -44,23 +44,43 @@
     "provider_class, test_data_files, result_parse_files",
     [
         # GenericProvider
-        (GenericProvider, [("ical", GENERIC_ICAL_DATA_PATH),], [GENERIC_ICAL_RESULT_PATH,],),
+        (
+            GenericProvider,
+            [
+                ("ical", GENERIC_ICAL_DATA_PATH),
+            ],
+            [
+                GENERIC_ICAL_RESULT_PATH,
+            ],
+        ),
```

Comment:
This is a net loss in readability, IMHO. Can we just strip out some trailing commas to make the newer version of Black happy with the one-liners instead, e.g.:

```suggestion
        (GenericProvider, [("ical", GENERIC_ICAL_DATA_PATH)], [GENERIC_ICAL_RESULT_PATH]),
```

or:

```suggestion
        (
            GenericProvider,
            [("ical", GENERIC_ICAL_DATA_PATH)],
            [GENERIC_ICAL_RESULT_PATH],
        ),
```

here and below?

Reply:
`black` is adding these commas